src/main/java/at/hannibal2/skyhanni/data/PetAPI.kt

```diff
@@ -1,14 +1,26 @@
 package at.hannibal2.skyhanni.data
 
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
 import at.hannibal2.skyhanni.utils.StringUtils.matches
 import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
 
 object PetAPI {
-    private val petMenuPattern by RepoPattern.pattern(
-        "misc.pet.menu.title",
+    private val group = RepoPattern.group("misc.pet")
+    private val petMenuPattern by group.pattern(
+        "menu.title",
         "Pets(?: \\(\\d+/\\d+\\) )?"
     )
 
+    private val petItemName by group.pattern(
+        "item.name",
+        "§.\\[Lvl (?<level>\\d)] (?<name>.*)"
+    )
+
+    private val neuRepoPetItemName by group.pattern(
+        "item.name.neu.format",
+        "(§f§f)?§7\\[Lvl 1➡(100|200)] (?<name>.*)"
+    )
+
```

Comment:
```suggestion
    private val patternGroup = RepoPattern.group("misc.pet")
    private val petMenuPattern by patternGroup.pattern(
        "menu.title",
        "Pets(?: \\(\\d+/\\d+\\) )?"
    )
    private val petItemName by patternGroup.pattern(
        "item.name",
        "§.\\[Lvl (?<level>\\d)] (?<name>.*)"
    )
    private val neuRepoPetItemName by patternGroup.pattern(
        "item.name.neu.format",
        "(§f§f)?§7\\[Lvl 1➡(100|200)] (?<name>.*)"
    )

``` To better match the format of patterns in the rest of the mod